code-challenges.js

```diff
@@ -33,22 +52,31 @@ const temp3 = 212
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
-
+// Pseudo code: 
+// The best way to get the length of an Array is to use the .length property. We can also assign a variable to that .length. We also need to use the .concat method to combine the two arrays.
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+const padres1998WorldSeries = padres1984WorldSeries.concat(padres1998WorldSeries.length)
+
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: 
+// The .reverse method is used to reverse letters. I will ned to use console.log to view the output. The variable has a name of "currentCohort" and it needs the .reverse in order to perform the action:
+console.log(currentCohort.reverse)
```

Comment:
If I run it this way in the terminal it will come back with the console.log 'cannot access currentCohort before initialization because the variable is using const and the console.log is written above it. Also, .reverse is a method that works on arrays, I would look into a way to turn a string into and array and then trying to implement the .reverse method (: